packages/cmf/__tests__/sagas/component.test.js

```diff
@@ -16,18 +16,39 @@ describe('sagas.component', () => {
 		const gen = onSagaStart(testAction);
 
 		// then
-		expect(gen.next().value).toEqual(fork(saga));
+		expect(gen.next().value).toEqual(fork(saga, undefined));
 		expect(gen.next(task).value).toEqual(take(`${CONST.WILL_UNMOUNT_SAGA_STOP}_my-saga`));
 		expect(gen.next().value).toEqual(cancel(task));
 	});
+
+	it('should onSagaStart fork action.saga with params and wait for unmount to cancel', () => {
```

Comment:
should onSagaStart forks action.saga with params and waits for unmount cancelling*